backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Controllers/TemplateGroupController.java

```diff
@@ -84,5 +92,32 @@ public ResponseEntity<?> updateTemplateGroup(@PathVariable("id") Long id, @Reque
         }
         return new ResponseEntity<>(updatedTemplateGroup, HttpStatus.OK);
     }
+
+    @Operation(summary = "Set the template for an specific group")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Template group set", content = @io.swagger.v3.oas.annotations.media.Content(mediaType = "application/json")),
```

Comment:
Throughout this entire file we can observe an anomaly which is the Content swagger import.
Instead of calling it like that, import it and call `@Content`

Reply:
this is due to one of our entities being called content, causing trouble in some of the controllers